Project/VetAppointmentAPI/VetAppointmentAPI/VetAppointment.WebAPI/Controllers/BillingEntryController.cs

```diff
@@ -10,18 +13,20 @@ namespace VetAppointment.WebAPI.Controllers
     public class BillingEntryController : ControllerBase
     {
         private readonly IBillingEntryRepository billingEntryRepository;
-        private readonly IUserRepository userRepository;
-        private readonly IOfficeRepository officeRepository;
+        private readonly IUserRepository issuerRepository;
+        private readonly IUserRepository customerRepository;
```

Comment:
I'm not sure why we need to use two instances of user respository. Is there something impeding the use of a single repo?